routes/login.js

```diff
@@ -31,8 +31,7 @@ router.route("/data").post(async (req, res) =>
   {
     return res.status(400).render('Auth/login', { error: e });
   }
-    const newData = await logInFunctions.logIn(email.trim().toLowerCase(), pass);
-    

```

Comment:
Can you check for logInFunctions.logIn().  I guess there is no logIn  function in logInFunctions..